python/src/IntersectionEvent_doc.i.in

```diff
@@ -10,11 +10,12 @@ The occurrence of all the events is necessary for the system event to occur (par
 Parameters
 ----------
 coll : sequence of :class:`~openturns.RandomVector`
-    Collection of events
+    Collection of events, evaluated from left to right.
+    Ordering the least probable events first can help to avoid evaluating the last ones.
```

Comment:
```suggestion
    Ordering the least probable events first can help to avoid evaluating the last ones.
    In case the relative probabilities of the events are unknown,
    they can be ranked quickly with (for example) a FORM analysis.
```